src/proteus/dswx_hls.py

```diff
@@ -2249,18 +2249,20 @@ def parse_runconfig_file(user_runconfig_file = None, args = None):
     if 'dem_file' not in ancillary_ds_group:
         dem_file = None
     else:
-        dem_file = ancillary_ds_group['dem_file']
+        dem_file = os.path.basename(
+            ancillary_ds_group['dem_file'])
 
     if 'landcover_file' not in ancillary_ds_group:
         landcover_file = None
     else:
-        landcover_file = ancillary_ds_group['landcover_file']
+        landcover_file = os.path.basename(
+            ancillary_ds_group['landcover_file'])
 
     if 'worldcover_file' not in ancillary_ds_group:
         worldcover_file = None
     else:
-        worldcover_file = ancillary_ds_group[
-            'worldcover_file']
+        worldcover_file = os.path.basename(
+            ancillary_ds_group['worldcover_file'])
```

Comment:
Unfortunately, changing the filepath here prevented the code from finding the source file for copying to the scratch directory. This is a fatal issue.

Instead, one idea could be to simply change the filenames that are written to the output  files. There might be a third place, but at minimum this would mean changing the code to  in two other places:
1) when writing the metadata dict
2) when writing entries into the log file

For example, to fix part 1, could maybe update Lines 2385-2392 to be something like this:

```
    # input datasets
    dswx_metadata_dict['HLS_DATASET'] = hls_dataset
    dswx_metadata_dict['DEM_FILE'] = os.path.basename(dem_file) if dem_file else '(not provided)'
    dswx_metadata_dict['LANDCOVER_FILE'] = \
        os.path.basename(landcover_file) if landcover_file else '(not provided)'
    dswx_metadata_dict['WORLDCOVER_FILE'] = \
        os.path.basename(worldcover_file) if worldcover_file \
                                     else '(not provided)'
```

Reply:
Thank you so much, @nemo794 ! This issue was addressed in [dbf9125](https://github.com/nasa/PROTEUS/pull/28/commits/dbf9125226a5baa9aad168c821e2ddd99e39e182).